src/Components/SmartComponents/Reports/ReportsHelper.js

```diff
@@ -70,7 +70,7 @@ export function constructFilterParameters(filterParams) {
         publish_date: filterParams.publish_date,
         public_from: from,
         public_to: to,
-        rule_presence: filterParams.security_rule
+        rule_presence: filterParams.rule_presence
```

Comment:
Filtering by security rule presence doesn't reflect in the paragraph on the first page:
> This report includes CVEs with a CVSS base score of 0.0 - 10.0; published anytime

it should also say

> and has/has no security rule

I think this line might cause the absence of it

Reply:
I fixed the missing info on the first page. Yes you are right about the severity it's also fixed in another pr